g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/TranslationWithPatternAction.java

```diff
@@ -51,6 +52,12 @@
 	    */
 	   public APIResponseDTO getTransPattern(TranslationWithPatternDTO data) throws Exception {
 	      logger.info("begin getTransPattern");
+	      try {
+	    	  ParameterValidationUtility.validateTranslationWithPatternAPI(data);
+	      }catch(RuntimeException excep) {
```

Comment:
Don't use RuntimeException, it will cause thread exit.

Reply:
done